packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -137,6 +149,32 @@ export class CfnInclude extends core.CfnElement {
     return ret;
   }
 
+  /**
+   * Returns the CfnInclude instance for a nested stack.
+   * For a nested stack to be returned by this method, it must be specified in the CfnIncludeProps.
+   * @param logicalId the ID of the stack to retrieve, as it appears in its template.
+   */
+  public getNestedStackTemplate(logicalId: string): CfnInclude {
+    const ret = this.nestedStackTemplates[logicalId];
+    if (!ret) {
+      throw new Error(`NestedStack with logical ID '${logicalId}' was not found in the template`);
+    }
+    return ret;
+  }
+
+  /**
+   * Returns the NestedStack with name logicalId.
+   * For a nested stack to be returned by this method, it must be specified in the CfnIncludeProps.
+   * @param logicalId the ID of the stack to retrieve, as it appears in its template.
+   */
+  public getNestedStack(logicalId: string): core.NestedStack {
+    const ret = this.nestedStacks[logicalId];
+    if (!ret) {
+      throw new Error(`Nested Stack with logical ID '${logicalId}' was not found`);
+    }
+    return ret;
+  }
+
```

Comment:
I have an alternative proposal here.

Instead of 2 methods, `getNestedStackTemplate()` and `getNestedStack()`, how about we have one method, that gives you both back?


```ts
export interface IncludedNestedStack {
  public readonly stack: core.Stack;
  public readonly includedTemplate: CfnInclude;
}

  public getNestedStack(nestedStackName: string): IncludedNestedStack {
     // ...
  }
```